src/main/java/de/tum/in/www1/artemis/web/rest/BonusResource.java

```diff
@@ -103,8 +104,9 @@ private BonusExampleDTO calculateGradeWithBonus(BonusStrategy bonusStrategy, Dou
      * @return final grade and points with bonus
      */
     @GetMapping("courses/{courseId}/exams/{examId}/bonus/calculate-raw")
+    // TODO: Remove this once the @PreAuthorize is fixed
```

Comment:
What is currently broken with the `@PreAuthorize`? Do we have to fix something in Artemis code, or wait for some library update?

Reply:
It's not broken. The endpoint is limited to admins because the feature is not yet released. Therefore it's manually configured.
I adapted the TODO, hope it clears things up